partcad/src/partcad/part_factory_python.py

```
@@ -23,13 +23,14 @@ def __init__(
         config,
         can_create=False,
         python_version=None,
+        extension=".py",
```

Ruff (0.8.2)

Missing type annotation for function argument `extension`

(ANN001)